src/common/globals.css

```diff
@@ -7,3 +7,14 @@
     font-family: "Lato", sans-serif;
   }
 }
+.hexagon {
+  -webkit-clip-path: polygon(
+    25% 5%,
+    75% 5%,
+    100% 50%,
+    75% 95%,
+    25% 95%,
+    0% 50%
+  );
+  clip-path: polygon(25% 5%, 75% 5%, 100% 50%, 75% 95%, 25% 95%, 0% 50%);
+}
```

Comment:
I'm not sure about putting it here. Can you check if we can have it closer to the hexagon directory? I don't think we will reuse it anywhere else